src/CMakeLists.txt

```diff
@@ -575,14 +575,18 @@ install(CODE "
    )
 ")
 
+# Make sure to escape a bunch of special characters in the path before trying to use it as a
+# regular expression below.
+string(REGEX REPLACE "([][+.*()^])" "\\\\\\1" escaped_path "${CMAKE_CURRENT_SOURCE_DIR}/zeek")
```

Comment:
I assume this works, but I'm lost on why it needs six(!) backslashes! :-)

Reply:
From a ticket on gitlab where I took inspiration for this from:

> After CMake language processing, \\\\\\1 becomes \\\1.  The first \\ places a literal \ in the output.  The second \1 is the placeholder that gets replaced by the match.